keyboards/readme.md

```diff
@@ -37,6 +37,7 @@ These keyboards are part of the QMK repository, but their manufacturers are not
 * [Arrow Pad](/keyboards/handwired/arrow_pad) &mdash; A custom creation by IBNobody.
 * [Atreus](/keyboards/atreus) &mdash; Made by Technomancy.
 * [Bantam44](/keyboards/bantam44) &mdash; It is a 44-key 40% staggered keyboard.
+* [CRBN](/keyboards/crbn) &mdash; A carbon fibre 40% ortho kit from Polarity Works.
```

Comment:
Just a note here that the number of community supported keyboards has by this point grown far beyond what would be reasonable to maintain in this list here, so it will likely be removed in favour of a direct link to the `keyboards/` directory, or https://qmk.fm/keyboards.
```suggestion
```